frontend/src/components/explorer/mapViewer/DataOverlay.vue

```diff
@@ -211,24 +211,25 @@ export default {
     }),
     filteredDataSourcesIndex() {
       if (this.$route.name === 'interaction-details') {
-        // do not include fluxomics data for the interaction partners page
-        const { fluxomics, ...dataSourcesIndex } = this.dataSourcesIndex;
+        // do not include reactionType data for the interaction partners page
+        const { reactionType, ...dataSourcesIndex } = this.dataSourcesIndex;
         return dataSourcesIndex;
       }
       return this.dataSourcesIndex;
     },
     filteredDataTypes() {
-      // Do not show fluxomics data for the interaction partners page
+      // Do not show reactionType data for the interaction partners page
       // The data type may still be selected, but not shown
       if (this.$route.name === 'interaction-details') {
-        const dataTypes = this.dataTypes.filter(elem => elem.name !== 'fluxomics');
+        const dataTypes = this.dataTypes.filter(elem => elem.name !== 'reactionType');
         return dataTypes;
       }
       return this.dataTypes;
     },
   },
   async created() {
     await this.getDataSourcesIndex(this.model.short_name);
+    this.DATA_TYPES_COMPONENTS = DATA_TYPES_COMPONENTS;
```

Comment:
The idiomatic Vue way of doing this is to add `DATA_TYPES_COMPONENT` to `data()`. In other words add `DATA_TYPES_COMPONENTS,` after `showModal: false,`.

Reply:
This was resolved through [4a44ab2](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1181/commits/4a44ab256970a9a37ce7235905d3c3c5139d0c8f).